scripts/bert/index.rst

```diff
@@ -45,4 +45,68 @@ It gets validation accuracy of `88.7% <https://raw.githubusercontent.com/dmlc/we
 It gets RTE validation accuracy of `70.8% <https://raw.githubusercontent.com/dmlc/web-data/master/gluonnlp/logs/bert/finetuned_rte.log>`_
 , whereas the the original Tensorflow implementation give evaluation results 66.4%.
 
-Some other tasks can be modeled with `--task_name` parameter.
\ No newline at end of file
+Some other tasks can be modeled with `--task_name` parameter.
+
+BERT for Sentence or Tokens Embedding
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The goal of this BERT Embedding is to obtain the sentence and token embedding from BERT's pre-trained model. In this way, instead of building and do fine-tuning for an end-to-end NLP model, you can build your model by just utilizing the sentence or token embeddings.
+
+Usage
+"""""
+
+.. code-block:: python
+    from bert.embedding import BertEmbedding
+
+    bert_abstract = """We introduce a new language representation model called BERT, which stands for Bidirectional Encoder Representations from Transformers.
```

Comment:
This example is great. Two potential improvements to make it even easier to use:
- provide a command line which takes a single sentence as input and output the embedding. For example: 
```
python embedding.py --sentence "BERT embedding is powerful."

Text: BERT embedding is powerful
Embedding:
array([-0.835946  , -0.4605566 , -0.95620036, ..., -0.95608854,
         -0.6258104 ,  0.7697007 ], dtype=float32)

```
- store this paragraph in a separate text file, and load this file in `embedding.py` with `TextLineDataset` and output the embeddings
```
python embedding.py --file ./sample_text.txt

Text:  We introduce a new ... 
Embedding:
array([-0.835946  , -0.4605566 , -0.95620036, ..., -0.95608854,
         -0.6258104 ,  0.7697007 ], dtype=float32)

Text: Unlike recent language representation ...
Embedding: 
```

If we make embedding.py an runnable example, we can add some test case in https://github.com/dmlc/gluon-nlp/blob/master/scripts/tests/test_scripts.py#L141 to make sure the code is not broken

Reply:
This improvement has been added in 592af32300f6b2bd65daa041b3f01eb84f41c017